src/NUnitFramework/framework/Internal/PropertyNames.cs

```diff
@@ -133,6 +133,10 @@ public class PropertyNames
         /// </summary>
         public const string IgnoreUntilDate = "IgnoreUntilDate";
 
+        /// <summary>
+        /// The optional Order the test will run in
+        /// </summary>
+        public const string Order = "Order";
         #endregion
```

Comment:
Aren't we still storing the order data in the tests? Or should create a new property on the work item for it you reckon?